### PR TITLE
Added metrics and removed retryHandler

### PR DESCRIPTION
**Pull request by patduin**
### :link: Related Issues

### CODE REVIEW

File: waggle-dance-integration-tests/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <Configuration>
   <Appenders>
     <Console name="STDOUT" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{ISO8601} %tn %-5p %c:%L - %m%n"/>
+      <PatternLayout pattern="%d{ISO8601} %t %-5p %c:%L - %m%n"/>
```
Comment (javsanbel2) [inline waggle-dance-integration-tests/src/test/resources/log4j2.xml:20]: What is the change here?

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java [hunk 2]
```diff
@@ -31,12 +31,15 @@
...
-
 import com.hotels.bdp.waggledance.conf.WaggleDanceConfiguration;
 
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
 @Component
 public class TTransportMonitor {
 
+  static final String METRIC_NAME_OPEN_TRANSPORTS = "open_transports_gauge";
```
Comment (javsanbel2) [inline waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java:42]: In this class you have a metric base name - (https://github.com/ExpediaGroup/waggle-dance/blob/main/waggle-dance-extensions/src/main/java/com/hotels/bdp/waggledance/extensions/client/ratelimit/RateLimitMetrics.java#L24)

Are you gonna add a base metric name also here to align with all metrics?

Comment (patduin) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java:42]: yeah I should I was testing out how it looks with the prefix, but I'll add a namespace

File: waggle-dance-integration-tests/src/test/resources/log4j2.xml (hunk 1 above)
Comment (patduin) [reply waggle-dance-integration-tests/src/test/resources/log4j2.xml:20]: It printed an "n" after the thread name, I think this was just a typo :)

File: waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java (hunk 2 above)
Comment (patduin) [reply waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/TTransportMonitor.java]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+## [3.13.3] - 2024-10-01
```
Comment (javsanbel2) [inline CHANGELOG.md:1]:
```suggestion
## [3.13.3] - 2024-10-02
```

**Review by javsanbel2 — APPROVED** [review]

**Review by mollonado — APPROVED** [review]

**Pull request merged**